README.md

```diff
@@ -202,6 +202,91 @@ Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL
 
     > **Note:** that every minute on the clock a job is triggered to reconcile the CRs with the chart values. These cronjobs are deleted when their respective features (e.g. `rhdhOperator.enabled=false`) are removed or when the chart is removed. This is required because the CRs are not managed by helm due to the CRD dependency pre availability to the deployment of the CR.
 
+### Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, ie: not in-memory.
+
+Kafka perfectly fullfil this reliability need.
+
+For that, you will need:
```

Comment:
Since the entire orchestrator chart is designed for OCP (there is another one for k8s), I'd try to remain as closer as  possible to the formal OCP admin guides.
Are the following steps fits with the recommendation in the OCP admin guide?

Reply:
Is that better?

Reply:
I was referring only to the steps to configure Kafka as a broker of knative eventing.

Reply:
ah, ok
So, is that better now?

Reply:
My intention was to provide a link to https://docs.openshift.com/serverless/1.33/eventing/brokers/kafka-broker.html which is the official admin guide for openshift serverless instead of relying on the upstream doc.
Mainly since we are opinionated about the version of the openshift serverless operators installed by the chart.

Reply:
I added the link to this doc in step 2 when we create the broker

But I think we still need to install the eventing resources we install at step 1, not sure if everything is installed with the operator